packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/use-navigation-menu-handlers.js

```diff
@@ -65,8 +65,10 @@ function useSaveNavigationMenu() {
 		};
 	}, [] );
 
-	const { editEntityRecord, saveEditedEntityRecord } =
```

Comment:
Question: should we rename `useSaveNavigationMenu` to `useUpdateNavigationMenu`? Really we should have 

`create` and `update` actions which both defer to a low level `save` action under the hood. This usage of `save` in actions which have varying behaviour is quite confusing.

Reply:
I think doing create and update inside save is a common DX pattern, and also a matter of personal taste? Ideally we'd have all three available 🤷🏻